docs/kh2/file/type/mset.md

```diff
@@ -0,0 +1,81 @@
+# [Kingdom Hearts II](../../index.md) - MSET (Moveset)
+
+The MSET files contain animations and the effects related to them (I-frames, hitboxes, effects...)
+It is a [BAR file](bar.md) that contains a list of "slots". Each slot may or may not have a "motion".
+The various actions in the game point to a specific motion by a slot position in this list, thus many of the motions are dummies, since the characters only use some of them.
+Eg: For Sora, slot 0 is idle, slot 4 is walking and slot 604 is the basic attack.
+
+Each motion is a BAR file with 2 entries, the model animation and the effects.
+
+## Animation structure
+
+???
+
+## Effects structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 |  Header
+| X 	 |  Type A effect (Happens during X frames)
```

Comment:
It would be good to specify this is an array with the size of "Header -> Effect A Count" and that it points to #type-a-effect as a link too. I had to jump back and forward to start to understand what the document was talking about and to figure out the actual length of a field.

Reply:
I changed 1 to "single" and X to " array", hopefully that's more readable?
About the size, 2 thoughts of mine:
1. If you want to know the size, you can just go to the effect/entry and see how long it is. Eg: Type A's last data is an array of shorts in the offset 6, so the length of an A effect is 6 + 2 * (param size)
2. I can't define a specific length, since it depends on the parameter size of each effect.
That being said, I could add another column with the size in bytes if it's preferred (Eg. 6 + 2 * (param size))